### ISSUE TITLE
false positive flags superclass method as undefined

### ISSUE THREAD

**Issue by rjwalters** [python/mypy#17322]
**Bug Report**

mypy incorrectly complains that the `__setstate__` method is undefined in the `multiprocessing.Queue`

Here is where `__setstate__` is defined:

https://github.com/python/cpython/blob/0480fd628a33351be868460c6febc09541ac722c/Lib/multiprocessing/queues.py#L62

**To Reproduce**

https://mypy-play.net/?mypy=latest&python=3.12&flags=verbose%2Ccheck-untyped-defs&gist=31e5a18f3d712b3e9991216bb8013bd7

**Expected Behavior**

I expected no typechecking errors.

**Actual Behavior**

If you run the included gist above you will see:

``` shell
main.py:12: error: "__setstate__" undefined in superclass  [misc]
Found 1 error in 1 file (checked 1 source file)
```

**Comment by pajod** [comment]
Appears fixed as of 1.20
